Makefile

```diff
@@ -11,26 +11,31 @@ BUILD_FLAGS   ?= -v
 LDFLAGS       ?= -X main.version=$(VERSION) -X main.buildstamp=$(shell date -u '+%Y-%m-%d_%I:%M:%S%p') -X main.githash=$(shell git rev-parse HEAD) -w -s
 
 
-default: build.local
+default: help
```

Comment:
Help should IMO not be default. The convention is you can just run `make` to build a program.